sdk/resourcemanager/azure-resourcemanager-samples/pom.xml

```diff
@@ -143,6 +143,11 @@
       <version>5.6.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-api;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
The change is required due to `docker-java` we used have [com.google.code.findbugs.annotation](https://github.com/docker-java/docker-java/blob/master/docker-java/pom.xml#L102-L107) as provided scope.